packages/react/src/components/AudioPlayer/README.stories.mdx

```diff
@@ -0,0 +1,124 @@
+import contributing from '../../../../../docs/contributing-license.md';
+import AudioPlayer from './AudioPlayer';
+
+# Audio Player
+
+> Audio players play embedded audios using the Kaltura video platform. It can be
+> used in inline patterns. It is always the full width of its
+> containing element and maintains a fixed height of 48 pixels.
+
+> 💡 Check our
+> [CodeSandbox](https://githubbox.com/carbon-design-system/carbon-for-ibm-dotcom/tree/master/packages/react/examples/codesandbox/components/audioplayer)
+> example implementation.
+
+[![Edit @carbon/ibmdotcom-web-components](https://codesandbox.io/static/img/play-codesandbox.svg)](https://githubbox.com/carbon-design-system/carbon-for-ibm-dotcom/tree/master/packages/react/examples/codesandbox/components/audioplayer)
+
+## Getting started
+
+Here's a quick example to get you started.
+
+##### CSS
+
+```css
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+@import '@carbon/ibmdotcom-styles/scss/components/audio-player/audio-player';
+```
+
+> 💡 Only import fonts once per usage
+
+##### JS
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { AudioPlayer } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+function App() {
+  return (
+    <AudioPlayer
+      videoId="your-video-id"
+    />
+  );
+}
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+Add the following line in your `.env` file at the root of your project.
+[See more details](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/tree/master/packages/styles#usage).
+
+```
+  SASS_PATH=node_modules:src
+```
+
+> 💡 Don't forget to import the AudioPlayer styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/master/packages/styles).
+
+## Environment Variables
+
+The partner ID and uiconf ID necessary to render videos from the Kaltura can be
+set from the `.env` file.
+
+example:
+
+```
+KALTURA_PARTNER_ID=243342
+KALTURA_UICONF_ID=12905712
+```
+
+## Manual usage of Kaltura API
+
+With `@carbon/ibmdotcom-services`, you can keep track of the readiness of
+[Kaltura API](http://player.kaltura.com/docs/api) and use it when it is ready.
+Here is an example of stopping the video player when it is hidden in DOM:
+
+```javascript
+import KalturaPlayerAPI from '@carbon/ibmdotcom-services/es/services/KalturaPlayer/KalturaPlayer';
+
+// Keeps track of element resize and detect if the element of our interest is hidden
+const resizeObserver = new ResizeObserver(entries => {
+  entries.forEach(entry => {
+    const { width, height } = entry.contentRect;
+    if (width === 0 && height === 0) {
+      // Uses Kaltura API to stop the video
+      entry.target.sendNotification('doPause');
+    }
+  });
+});
+
+VideoPlayerAPI.checkScript().then(() => {
```

Comment:
```suggestion
KalturaPlayerAPI.checkScript().then(() => {
```

Reply:
There is another code error in the examples on line 44~46 that I'm fixing
```
<AudioPlayer
  videoId="your-video-id"
/>
```